virtualization/windowscontainers/quick-start/Insider-Known-Issues.md

```diff
@@ -0,0 +1,16 @@
+# Known Issues for Insider Builds
+
+## Build 16237
+
+- Hyper-V Containers are not working properly. This workaround is needed to use Hyper-V Containers in build 16237. Run these commands in PowerShell:
+
+```PowerShell
+Get-ComputeProcess | ? IsTemplate -eq $true | Stop-ComputeProcess -Force
+Set-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Virtualization\Containers\' -Name TemplateVmCount -Type dword -Value 0 -Force
+```
+
+- Nano Server now runs as user, so commands that require administrator priviliges will fail. Including a line such as "RUN setx /M PATH" will cause the build to fail. For this scenario, you can use this alternative:
```

Comment:
In my opinion the `docker build` should run in root = ContainerAdministrator mode as we cannot specify a user while building. So a PATH can be set globally.
If we only can add PATH for current user it is impossible to switch users, eg. running the container as ContainerAdministrator later on having that PATH.
WDYT?